packages/eslint-plugin/src/configs/react.js

```diff
@@ -273,15 +313,24 @@ const config = {
      * @see https://github.com/import-js/eslint-plugin-import
      */
     'import/no-extraneous-dependencies': ['error', { devDependencies: false }],
+    'import/no-duplicates': 'warn',
```

Comment:
new rules 
- handles sorting imports by path
- handles duplicate path imports
- enforces to have imports always at the top of the file ( default behaviour of ECMAscript anyways - imports are hoisted )